app/graphql/queries/consultation_response/venter_map.rb

```diff
@@ -0,0 +1,13 @@
+module Queries
+	module ConsultationResponse
+		class VenterMap < Queries::BaseQuery
+	    description "Get resonse keyword map for consultation responses"
+	    type GraphQL::Types::JSON, null: false
+
+	    def resolve
+	    	hash_map_response = Rails.cache.read("response_hash_map")
```

Comment:
So, what happens when let's say the cache is wiped out and no responses are added. The `read` method will keep returning `nil` and then the cache will have to be manually created.